src/com/epam/javatrainings/iohomework1/Task1.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.javatrainings.iohomework1;
+
+import java.io.*;
+import java.net.URI;
+
+public class Task1 {
+    public static void main(String[] args) {
+        URI u = URI.create("https://www.google.com/");
+        copyingByte_By_Byte(u);
+        copyingViaBuffer(u);
+    }
+    public static void copyingByte_By_Byte(URI u){
```

Comment:
I'd create AbstractPageDownloader, then have 2 implementations. If you think it is overkill just create PageDownloader class and have there 2 methods